recipes/sspace_basic/build.sh

```diff
@@ -1,7 +1,22 @@
 #!/bin/bash
 
-mkdir -p $PREFIX/bin && cp -r bin/* $PREFIX/bin && cp SSPACE_Basic.pl $PREFIX/bin && cp -r tools/* $PREFIX/bin
-chmod +x $PREFIX/bin/*
-sed -i.bak 's|/usr/bin/perl|/usr/bin/env perl|' $PREFIX/bin/*.pl
-ln -s $PREFIX/bin/SSPACE_Basic.pl $PREFIX/bin/sspace_basic
-mkdir -p $PREFIX/bin/dotlib && cp -r dotlib/* $PREFIX/bin/dotlib
\ No newline at end of file
+# Copying the tools bin 
+mkdir -p $PREFIX/bin
+mkdir -p $PREFIX/bin/bin
+cp -r bin/* $PREFIX/bin
+cp -r bin/* $PREFIX/bin/bin  # Tool hardcoded the path of other used scripts to $Bin/bin/<tool>.pl
```

Comment:
This looks weird.
Could you please explain again why `../bin/bin/` is really needed ?

Reply:
In order for the tool to work, it needs two things:
[- It's custom `$Bin/dotlib` library. ](https://github.com/nsoranzo/sspace_basic/blob/4fe5c275c94b36e02d1b69438a2f7e022ecb58bc/SSPACE_Basic.pl#L93)
[- Custom scripts in `$Bin/bin`. ](https://github.com/nsoranzo/sspace_basic/blob/4fe5c275c94b36e02d1b69438a2f7e022ecb58bc/SSPACE_Basic.pl#L284)

Here `$Bin` is determined at runtime by [checking the location](https://github.com/nsoranzo/sspace_basic/blob/4fe5c275c94b36e02d1b69438a2f7e022ecb58bc/SSPACE_Basic.pl#L92) from which the main tool `sspace_basic` is ran. 
```
use FindBin qw($Bin);
```

Because we export this tool to `$PREFIX/bin/` it will look for it's custom scripts in `$PREFIX/bin/bin`.  An alternative would be to include another sed statement where we replace `$Bin/bin` with nothing. But doesn't seem ideal. 

Let me know if you have any other suggestions on how to fix this.

Reply:
Thanks!